src/test/java/net/dv8tion/jda/test/entities/guild/SystemChannelFlagTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.test.entities.guild;
+
+import net.dv8tion.jda.api.entities.guild.SystemChannelFlag;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static net.dv8tion.jda.api.entities.guild.SystemChannelFlag.*;
+
+import java.util.EnumSet;
+
+/**
+ * Tests the class {@link SystemChannelFlag}, especially the conversion logic
+ * between its {@link EnumSet} and bitmask representations.
+ */
+public final class SystemChannelFlagTest
+{
+    @Test
+    public void testFromBitmask()
```

Comment:
I think these tests aren't really useful, we can probably live without them since they will likely never be updated if new flags are added.